src/index.js

```diff
@@ -0,0 +1,150 @@
+ * @param {string} state current Meridiem state
+ */
+function changeTopScroll (currentCity, hour, state) {
+  const topContainer = document.getElementsByClassName('top-end')[0]
+  topContainer.innerHTML = ''
+  for (let i = 0; i < 5; i += 1) {
+    if (i === 0) {
+      const element = document.createElement('div')
+      element.classList.add('topscroll-items')
+      element.innerHTML = `
+      <div class="item-time">NOW</div>
+      <div class="pipe">|</div>
+      <div class="scroll-icon">
+          <img alt="weathericon" class="top-ic"  >
+      </div>
+      <div> ${currentCity.temperature} </div>
+      `
+      topContainer.appendChild(element)
+      changeIconImage(currentCity.temperature, i)
+    } else {
+      const element = document.createElement('div')
+      element.classList.add('topscroll-items')
+      if (parseInt(hour) === 11) hour = 12
+      else hour = (parseInt(hour) + 1) % 12
+      if (hour === 1) {
+        if (state === 'PM') state = 'AM'
+        else state = 'PM'
+      }
+      console.log(hour)
+      element.innerHTML = `<div class="item-time">${hour.toString() + state}</div>
+      <div class="pipe">|</div>
+      <div class="scroll-icon">
+          <img alt="weathericon" class="top-ic">
+      </div>
+      <div> ${currentCity.nextFiveHrs[i - 1]} </div>`
+      topContainer.appendChild(element)
+      changeIconImage(currentCity.nextFiveHrs[i - 1], i)
+    }
+
+    if (i < 4) {
+      const element = document.createElement('div')
+      element.classList.add('topscroll-items')
+      element.innerHTML = `
+    <div class="item-time"></div>
+    <div class="pipe">|</div>
+    <div class="scroll-icon"></div>
+    <div class="num"></div> `
+      topContainer.appendChild(element)
+    }
+  }
+}
+
+/**
+ *This function will change values if the option is changed in city selection
+ */
+function changeValues () {
+  const selectedCity = document.getElementsByClassName('option-select')[0].value.toLowerCase()
+  console.log(selectedCity)
+  document.getElementById('top-img').src = '../Assets/HTML&CSS/Icons_for_cities/' + selectedCity + '.svg'
+  console.log(allCityDetails[selectedCity].temperature)
+  document.getElementById('tempc-val').innerHTML = allCityDetails[selectedCity].temperature
+  document.getElementById('tempf-val').innerHTML = ((parseInt(allCityDetails[selectedCity].temperature) * 9 / 5) + 32).toFixed(2) + 'F'
+  document.getElementById('humid-val').innerHTML = allCityDetails[selectedCity].humidity
+  document.getElementById('precip-val').innerHTML = allCityDetails[selectedCity].precipitation
+  const time = document.getElementsByClassName('time-text')[0]
+  const date = document.getElementById('date-text')
+  const timeJson = allCityDetails[selectedCity].dateAndTime
+  console.log(timeJson)
+  const separatedArray = timeJson.split(', ')
+  console.log(separatedArray)
+  const months = ['JAN', 'FEB', 'MAR', 'APR', 'MAY', 'JUN', 'JUL', 'AUG', 'SEP', 'OCT', 'NOV', 'DEC']
```

Comment:
Let's store this type of constant in a separate constant file and import and use it.